bin/travis/before_script.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname ${BASH_SOURCE[0]})/../../"
+
+node_modules/.bin/sequelize --config /etc/blocktogether/sequelize.json db:migrate
```

Comment:
Can you put a `./` in from of this and `run-dev.sh`? I generally like to do that in case `.` isn't in PATH, which is fairly common. I'm actually surprised it is in Travis' PATH.